requirements/requirements-dev.txt

```diff
@@ -573,15 +577,19 @@ twisted[tls]==21.2.0
     #   treq
 typed-ast==1.4.0
     # via
+    #   astroid
     #   black
     #   mypy
 typing-extensions==3.7.4.3
     # via
     #   -r requirements.txt
     #   black
+    #   importlib-metadata
     #   matrix-synapse
     #   mypy
     #   signedjson
+    #   structlog
+    #   web3
```

Comment:
What's with the commented out dependencies?

Reply:
Those explain the dependency tree (that's why they say `via`).